src/components/Background/index.js

```diff
@@ -57,7 +42,7 @@ export default class Background extends Component {
           autoPlay
           loop
           playsInline
-          preload
+          preload="true"
```

Comment:
This should be `preload="auto"`
[https://www.w3schools.com/tags/tag_video.asp](https://www.w3schools.com/tags/tag_video.asp)